src/pages/Transfer/TransferOverview/components/TransferForm/TransferForm.tsx

```diff
@@ -0,0 +1,158 @@
+import { CurrencyExt, newMonetaryAmount } from '@interlay/interbtc-api';
+import { mergeProps } from '@react-aria/utils';
+import { Key, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+
+import { StoreType } from '@/common/types/util.types';
+import {
+  convertMonetaryAmountToValueInUSD,
+  displayMonetaryAmount,
+  displayMonetaryAmountInUSDFormat,
+  newSafeMonetaryAmount
+} from '@/common/utils/utils';
+import { Dd, DlGroup, Dt, Flex, Input, TokenInput } from '@/component-library';
+import { AuthCTA } from '@/components';
+import { GOVERNANCE_TOKEN, TRANSACTION_FEE_AMOUNT } from '@/config/relay-chains';
+import { isFormDisabled, useForm } from '@/lib/form';
+import {
+  TRANSFER_AMOUNT_FIELD,
+  TRANSFER_RECIPIENT_FIELD,
+  TRANSFER_TOKEN_FIELD,
+  TransferFormData,
+  transferSchema
+} from '@/lib/form/schemas';
+import { getTokenPrice } from '@/utils/helpers/prices';
+import { useGetBalances } from '@/utils/hooks/api/tokens/use-get-balances';
+import { useGetCurrencies } from '@/utils/hooks/api/use-get-currencies';
+import { useGetPrices } from '@/utils/hooks/api/use-get-prices';
+import { Transaction, useTransaction } from '@/utils/hooks/transaction';
+import { useSelectCurrency } from '@/utils/hooks/use-select-currency';
+
+import { StyledDl } from './TransferForm.styles';
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+type TransferFormProps = {};
+
+// eslint-disable-next-line no-empty-pattern
+const TransferForm = ({}: TransferFormProps): JSX.Element => {
+  const { t } = useTranslation();
+  const { bridgeLoaded } = useSelector((state: StoreType) => state.general);
+
+  const prices = useGetPrices();
+  const { getCurrencyFromTicker } = useGetCurrencies(bridgeLoaded);
+  const { getBalance, getAvailableBalance } = useGetBalances();
+  const { items: selectItems } = useSelectCurrency();
+
+  const [transferToken, setTransferToken] = useState<CurrencyExt>(GOVERNANCE_TOKEN);
+
+  const transaction = useTransaction(Transaction.TOKENS_TRANSFER, {
+    onSuccess: () => {
+      form.resetForm();
+    }
+  });
+
+  const transferTokenBalance = transferToken && getAvailableBalance(transferToken.ticker);
+
+  const governanceBalance = getBalance(GOVERNANCE_TOKEN.ticker)?.free || newMonetaryAmount(0, GOVERNANCE_TOKEN);
+  const minAmount = transferToken && newMonetaryAmount(1, transferToken);
```

Comment:
Bit confused by this—are you setting the min transferable amount to 1?

Reply:
So that the user cannot transfer 0 amounts

Reply:
Yes, got that bit :) what I mean is what about values between 0 and 1?

Reply:
No this amount is 0.0000000000000001. Its like the lower unit possible

Reply:
Ok, think we might have some issues elsewhere in our code then :) we have a lot of instances of newMonetaryAmount being called with values less than 1, so let's review and make sure everything's as it should be.

Well outside the scope of this PR though, so this can be resolved.

Reply:
Actually you can completely ignore that last comment—I've double-checked the other instances, and I'd forgotten about the base parameter. All looks good, complete red herring :)